services/consensuscontext/validate_transactions_block_test.go

```diff
@@ -85,23 +83,167 @@ func TestTransactionsBlockValidators(t *testing.T) {
 		require.Equal(t, ErrMismatchedPrevBlockHash, errors.Cause(err), "validation should fail on incorrect prev block hash", err)
 	})
 
-	t.Run("should return error for transaction block with failing tx ordering validation", func(t *testing.T) {
```

Comment:
this is not here anymore since you moved the ordering validation to a separate step, right?

Reply:
yes